pkg/memory-monitor/Makefile

```
@@ -0,0 +1,105 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+CC=gcc
+CFLAGS=-Wall -static -O3
```

```suggestion
CFLAGS=-Wall -static -O2
```
`O3` might cut corners that we're not sure about, especially if the code lacks tests.

---

Ok, I'll change it.

---

Fixed